ui/src/alerting/components/builder/CheckMatchingRulesCard.tsx

```diff
@@ -1,33 +1,67 @@
 // Libraries
 import React, {FunctionComponent, useState, useEffect} from 'react'
 import {connect} from 'react-redux'
+import {uniq} from 'lodash'
+import {fromFlux} from '@influxdata/giraffe'
 
 // Components
 import MatchingRuleCard from 'src/alerting/components/builder/MatchingRuleCard'
-import {SpinnerContainer, TechnoSpinner} from '@influxdata/clockface'
+import {
+  SpinnerContainer,
+  TechnoSpinner,
+  FlexBox,
+  FlexDirection,
+  AlignItems,
+} from '@influxdata/clockface'
 
 // API
-import * as api from 'src/client'
+import {getNotificationRules} from 'src/client'
```

Comment:
Did we establish that we'd follow an aliased convention of `apiGetNotificationRules`?

Reply:
thanks for reminder! :)